pkg/virtualization/core/service/gpu.go

```diff
@@ -43,7 +43,7 @@ func (vmms *VirtualSystemManagementService) AttachGPU(vm *virtualsystem.VirtualM
 	return
```

Comment:
The error that is returned and what is being printed is different.

Can we return a proper error and remove these log statements